CourseApp/Formula10.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace Formula
+{
+    public class Formula10
+    {
+        public double Form10(double x, double a, double b)

```

Comment:
Bad name, method should be a verb

Reply:
@swbonjour Please - rename this one - to something like CalculateSingleValue or CalculateSinglePoint